*** setup.py	2013-02-26 01:46:32.851402490 +1100
--- ../setup.py	2013-02-26 01:47:16.403403772 +1100
***************
*** 46,52 ****
        author_email = __contact__,
        url = __url__,
        license = __license__,
!       packages  = ["txtorcon", "txtorcon.test"],
  #      scripts = ['examples/attach_streams_by_country.py'],
  
        ## I'm a little unclear if I'm doing this "properly", especially
--- 46,52 ----
        author_email = __contact__,
        url = __url__,
        license = __license__,
!       packages  = ["txtorcon", ],
  #      scripts = ['examples/attach_streams_by_country.py'],
  
        ## I'm a little unclear if I'm doing this "properly", especially
***************
*** 54,67 ****
        ## the first member of the tuple? Why does it seem I need to
        ## duplicate this in MANIFEST.in?
  
!       data_files = [('share/txtorcon', ['INSTALL', 'README', 'TODO', 'meejah.asc']),
  
                      ## this includes the Sphinx source for the
                      ## docs. The "map+filter" construct grabs all .rst
                      ## files and re-maps the path
!                     ('share/txtorcon', ['docs/apilinks_sphinxext.py', 'docs/conf.py', 'docs/Makefile'] + map(lambda x: os.path.join('docs', x), filter(lambda x: x[-3:] == 'rst', os.listdir('docs'))) + map(lambda x: os.path.join('docs/_static', x), os.listdir('docs/_static'))),
  
                      ## include all the examples
!                     ('share/txtorcon/examples', map(lambda x: os.path.join('examples', x), filter(lambda x: x[-3:] == '.py', os.listdir('examples'))))
                      ]
        )
--- 54,67 ----
        ## the first member of the tuple? Why does it seem I need to
        ## duplicate this in MANIFEST.in?
  
!       data_files = [#('share/txtorcon', ['INSTALL', 'README', 'TODO', 'meejah.asc']),
  
                      ## this includes the Sphinx source for the
                      ## docs. The "map+filter" construct grabs all .rst
                      ## files and re-maps the path
!                     #('share/txtorcon', ['docs/apilinks_sphinxext.py', 'docs/conf.py', 'docs/Makefile'] + map(lambda x: os.path.join('docs', x), filter(lambda x: x[-3:] == 'rst', os.listdir('docs'))) + map(lambda x: os.path.join('docs/_static', x), os.listdir('docs/_static'))),
  
                      ## include all the examples
!                     #('share/txtorcon/examples', map(lambda x: os.path.join('examples', x), filter(lambda x: x[-3:] == '.py', os.listdir('examples'))))
                      ]
        )
